lib/models/admin_model.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:badiup/models/user_model.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class Admin extends User {
+  Admin({String email, String name, RoleType role, DocumentReference setting, DateTime created})
```

Comment:
Line too long.

Reply:
@dragonfly91 Done by `flutter format lib/models/admin_model.dart`

Do you think we should format all our codes by `flutter format .`